src/Sylius/Bundle/ProductBundle/Form/DataTransformer/ProductsToProductAssociationsTransformer.php

```diff
@@ -175,7 +177,7 @@ private function setAssociatedProductsByProductCodes(ProductAssociationInterface
     /**
      * @param Collection|null $productAssociations
      */
-    private function setProductAssociations($productAssociations)
+    private function setProductAssociations(?Collection $productAssociations): void
     {
         $this->productAssociations = $productAssociations;
```

Comment:
Side note: it looks weird for a transformer to have state.

Reply:
Agreed 👍  However I would fix it in separate PR just to make this one as simple as possible.